source/user-manual/wazuh-dashboard/config-file.rst

```diff
@@ -1,14 +1,14 @@
 .. Copyright (C) 2015, Wazuh, Inc.
 
 .. meta::
-  :description: The Wazuh dashboard includes a configuration file where you can define custom values for several options. Learn more about it in this section.
+  :description: The Wazuh dashboard plugin includes a configuration file where you can define custom values for several options. Learn more about it in this section.
 
 .. _wazuh_dashboard_config_file:
 
 Configuration file
 ==================
 
-The Wazuh dashboard includes a configuration file located at ``/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml`` where you can define custom values for several options. This section describes all the settings available in this file.
+The Wazuh dashboard plugin includes a configuration file located at ``/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml`` where you can define custom values for several options. This section describes all the settings available in this file.
```

Comment:
"Wazuh dashboard plugin" isn't used in the documentation. The Wazuh components described in the Wazuh documentation are the server, the indexer, the dashboard, and the agent.

Reply:
We will need to address this in https://github.com/wazuh/internal-devel-requests/issues/391

I wanted to make this distinction because there are 2 relevant config files in Wazuh dashboard:

- `opensearch-dashboards.yml`
- `wazuh.yml`

As long as the full path is specified to make them easily to find, I don't mind using "Wazuh dashboard" always.

What are your thoughts in this?